media/main.js

```diff
@@ -209,6 +212,9 @@ $(document).ready(function () {
       case "author":
         authorName = message.author;
         break;
+      case "explanation":
+        console.log("message.explanation - main.js: ", message.explanation);
+        explanation = message.explanation;
```

Comment:
Where is this defined?

Reply:
Bad understanding of the code from my part. Moved stuff around. We're almost there.